extensions-contrib/opencensus-extensions/pom.xml

```diff
@@ -44,29 +44,21 @@
       <groupId>io.opencensus</groupId>
       <artifactId>opencensus-proto</artifactId>
       <version>0.2.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.druid</groupId>
```

Comment:
Dont we need this ?

Reply:
it's still there, the diff is just a bit wonky

Reply:
ok